Makefile

```diff
@@ -44,4 +51,4 @@ install-exclude:
 # target: install-systemd - Install systemd timer and service files
 install-systemd:
 	install -d $(DEST_SYSTEMD)
-	install -m 0644 $(SRCS_SYSTEMD) $(DEST_SYSTEMD)
+	install $(SRCS_SYSTEMD) $(DEST_SYSTEMD)
```

Comment:
I found this file permission to be redundant, since this is what I get by default on new files placed in this destination.